core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -17,11 +17,13 @@
  * Protractor tests.
  */
 
+import {element, browser, by} from 'protractor';
+
 var forms = require('./forms.js');
 var general = require('./general.js');
 var interactions = require('../../../extensions/interactions/protractor.js');
 var ruleTemplates = require(
-  '../../../extensions/interactions/rule_templates.json');
+  '../../../../extensions/interactions/rule_templates.json');
```

Comment:
Seems odd that this needs four `../` when the previous one needs three?

Reply:
This is because the js file will be fetched from `local_compiled_js/extensions` but the `json` will be fetched from `oppia/extensions`